mesonbuild/mesonlib/universal.py

```diff
@@ -1893,6 +1893,11 @@ def get_wine_shortpath(winecmd: T.List[str], wine_paths: T.Sequence[str]) -> str
 
     wine_paths = list(OrderedSet(wine_paths))
 
+    # do we even need to bother shortening the path?
+    wine_path = ";".join(wine_paths)
```

Comment:
I think we generally use single-quotes.

In general, this LGTM.

Reply:
Fixed, thanks. I'm used to `black`. :)